README.md

```diff
@@ -42,5 +45,7 @@ Then run:
 ```
 bundle
 bundle exec rake db:create
+bundle exec rake db:migrate
+bundle exec rake slack:update
```

Comment:
I will set Heroku Scheduler daily. This is the standard way to do cron-like task on Heroku.
https://elements.heroku.com/addons/scheduler